bin/install-local-tests.sh

```diff
@@ -1,60 +1,59 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -e
 
+# shellcheck disable=SC1091
+source "$(dirname "$0")/helpers.sh"
+
 # Initialize variables with default values
 TMPDIR="/tmp"
 DB_NAME="wordpress_test"
 DB_USER="root"
 DB_PASS=""
 DB_HOST="127.0.0.1"
-WP_VERSION="latest"
+WP_VERSION=${WP_VERSION:-latest}
 SKIP_DB=""
 
-# Display usage information
-usage() {
-  echo "Usage:"
-  echo "./install-local-tests.sh [--dbname=wordpress_test] [--dbuser=root] [--dbpass=''] [--dbhost=127.0.0.1] [--wpversion=latest] [--no-db]"
-}
-
 # Parse command-line arguments
-for i in "$@"
-do
-case $i in
+for i in "$@"; do
+  case $i in
     --dbname=*)
     DB_NAME="${i#*=}"
-    shift
     ;;
     --dbuser=*)
     DB_USER="${i#*=}"
-    shift
     ;;
     --dbpass=*)
     DB_PASS="${i#*=}"
-    shift
     ;;
     --dbhost=*)
     DB_HOST="${i#*=}"
-    shift
     ;;
-    --wpversion=*)
+    --version=*)
     WP_VERSION="${i#*=}"
-    shift
     ;;
-    --no-db)
+    --skip-db=*)
     SKIP_DB="true"
-    shift
+    ;;
+    --tmpdir=*)
+    TMPDIR="${i#*=}"
     ;;
     *)
     # unknown option
-    usage
+    echo "Unknown option: $i. Usage: ./bin/install-local-tests.sh --dbname=wordpress_test --dbuser=root --dbpass=root --dbhost=localhost --version=latest --tmpdir=/tmp --skip-db=true"
     exit 1
     ;;
-esac
+  esac
 done
 
 # Run install-wp-tests.sh
 echo "Installing local tests into ${TMPDIR}"
-bash "$(dirname "$0")/install-wp-tests.sh" "$DB_NAME" "$DB_USER" "$DB_PASS" "$DB_HOST" "$WP_VERSION" "$SKIP_DB"
+echo "Using WordPress version: ${WP_VERSION}"
+
+if [ -z "$SKIP_DB" ]; then
+  bash "$(dirname "$0")/install-wp-tests.sh" --version="$WP_VERSION" --tmpdir="$TMPDIR" --dbname="$DB_NAME" --dbuser="$DB_USER" --dbpass="$DB_PASS" --dbhost="$DB_HOST"
+else
+  bash "$(dirname "$0")/install-wp-tests.sh" bash "$(dirname "$0")/install-wp-tests.sh" --version="$WP_VERSION" --tmpdir="$TMPDIR" --dbname="$DB_NAME" --dbuser="$DB_USER" --dbpass="$DB_PASS" --dbhost="$DB_HOST" --skip-db=true
+fi
```

Comment:
if skip db is the only difference better to use arg array pattern described above and append conditionally